data/api/client-server/profile.yaml

```
@@ -67,163 +104,140 @@ paths:
+                  [Common Namespaced Identifier Grammar](/appendices/#common-namespaced-identifier-grammar).
       responses:
         "200":
-          description: The display name for this user.
+          description: The profile field value was retrieved.
           content:
             application/json:
               schema:
                 type: object
-                properties:
-                  displayname:
-                    type: string
-                    description: The user's display name if they have set one, otherwise not
-                      present.
+                additionalProperties:
+                  type: string
               examples:
                 response:
-                  value: {
-                    "displayname": "Alice Margatroid"
-                  }
+                  value: { "displayname": "Alice" }
         "403":
           x-addedInMatrixVersion: "1.12"
-          description: The server is unwilling to disclose whether the user exists and/or
-            has a display name.
+          description:
+            The server is unwilling to disclose whether the user exists and/or
+            has the specified profile field.
           content:
             application/json:
               schema:
                 $ref: definitions/errors/error.yaml
               examples:
                 response:
-                  value: {
-                    "errcode": "M_FORBIDDEN",
-                    "error": "Profile lookup is disabled on this homeserver"
-                  }
+                  value:
+                    {
+                      "errcode": "M_FORBIDDEN",
+                      "error": "Profile lookup is disabled on this homeserver",
+                    }
         "404":
-          description: There is no display name for this user or this user does not exist.
+          description: There is no profile field with this key for this user, or the user does not exist.
```

Do we need to report the error response body here? Do we distinguish errors for missing keys v.s. missing users?

---

We do not, but I'm following the pattern of the previous one that mentioned the error could be for either no field or no user.

---

(I've marked this as resolved to help me track what needs work, but feel free to reopen!)